charts/kargo/values.yaml

```
@@ -97,7 +97,8 @@ webhooks:
 ## the Kargo controller is running.
 ## @descriptionEnd
 ## @skip kubeconfigSecrets
-kubeconfigSecrets: {}
+kubeconfigSecrets:
+  {}
```

I appreciate the removal of errant spaces on what should have been blank lines (as on line 51, for instance), because that should be completely non-controversial, but this particular change and others like it seem like they're unnecessarily bundling unrelated enforcement of an opinion into the PR. It's easier to approve this PR without such changes.

```suggestion
kubeconfigSecrets: {}
```

---

blame "Format on Save" :)